src/components/Intercom/index.tsx

```diff
@@ -77,7 +79,8 @@ const Intercom = ({
     <div
       className={containerClass}
       style={{
-        ...styles.wrapper
+        ...styles.wrapper,
+        ...wrapperStyleProps
```

Comment:
In testing this I wasn't able to affect the position of the placeholder icon with the styles on the container.  

If I moved `...wrapperStyleProps` down to the `div` with `role="button"` below it worked.   Perhaps the styles would make more sense there?